internal/cf/client.go

```diff
@@ -135,21 +149,32 @@ func NewOrganizationClient(organizationName string, url string, username string,
 			delete(clientCache, identifier)
 			isInCache = false
 		}
+		if cacheEntry.expiresAt.Before(time.Now()) {
+			delete(clientCache, identifier)
+			isInCache = false
+		}
```

Comment:
The same coding should be also applied to **NewSpaceClient** and **NewSpaceHealthChecker**.

Reply:
good catch. Thxs. Is fixed.